www/js/utils.js

```diff
@@ -184,13 +186,24 @@ utils.factory('api', ['req', 'constants', function(req, constants) {
             req.delete(url, success, err);
         },
         getFilteredArticles: function(params, success, err) {
-            var url = constants.BASE_SERVER_URL + 'resources/find';
+		var key = $localStorage.get("leaderAPIKey");
+			console.log(key);
+            var url = constants.BASE_SERVER_URL + 'resources/find?LeaderAPIKey=' + key;
```

Comment:
Should produce an issue when grabbing the  resources if not a leader...  From looking at the resource code